guides/v2.3/comp-mgr/cli/cli-upgrade.md

```diff
@@ -10,128 +10,105 @@ functional_areas:
 {% capture ee %}{{site.data.var.ee}}{% endcapture %}
 -->
 
-Upgrade Magento from the command line if you installed the software using any of the following:
+You can upgrade Magento from the command line if you installed the software by:
 
-* Downloaded the {% glossarytooltip 7490850a-0654-4ce1-83ff-d88c1d7d07fa %}metapackage{% endglossarytooltip %} using `composer create-project`
-* Installed the compressed archive
+* Downloading the {% glossarytooltip 7490850a-0654-4ce1-83ff-d88c1d7d07fa %}metapackage{% endglossarytooltip %} using `composer create-project`.
+* Installing the compressed archive.
 
-There are two ways to upgrade your Magento application to 2.3:
-1. Manual: [Upgrade using the command line].
-2. Semi-automated: [Upgrade using the script].
-The upgrading scenario is the same from the system's point of view except that several steps are automated using the PHP script.
+There are two ways to upgrade your Magento application to the 2.3 version:
+1. [Upgrade using the command line] (manual process)
+2. [Upgrade using the script] (semi-automated process)
```

Comment:
Would it be helpful to mention why each one is best? The intros for each of these has a "best used by" message. Just a suggestion, not a required change.

Reply:
I think normally, yes. But in this case we don't want to encourage people to use one way over another (if they are unskilled and use script way, errors may incur which causes Support strain).  I'm getting conflicting info, between Dima and Peter, about which path we prefer users to take, so I'm going to leave this for now.